addons/mail/static/src/components/message/message.xml

```diff
@@ -173,12 +191,12 @@
                             <ul class="o_Message_trackingValues">
                                 <t t-foreach="trackingValues" t-as="value" t-key="value.id">
                                     <li>
-                                        <div class="o_Message_trackingValue">
+                                        <div class="o_Message_trackingValue" aria-atomic="true">
```

Comment:
Why adding `aria-atomic` here and only here? It don't know it but it looks related to area that are supposed to change, but this part is not really dynamic as tracking is purely static history.

Reply:
Changed by role="group". This is to make list items to be threated as a single element, instead of a multi-line section. This is because it contain several texts and an image.